pilot/docker/Dockerfile.proxytproxy

```diff
@@ -0,0 +1,43 @@
+FROM istionightly/base_debug
+ARG proxy_version
+ARG istio_version
+
+# Please keep in sync with Dockerfile.v2
+# This proxy will use TPROXY interception. Core dump won't work - see issue 5745
+# User will need to use the proxy annotation to use this image, as well as the
+# original InterceptionMode annotation to run as root and add NET_ADMIN
+
+# TODO: evaluate if we should add NET_ADMIN in all cases, to support future firewall
+# and dynamic iptable config
+
+ADD envoy /usr/local/bin/envoy
+
+ADD pilot-agent /usr/local/bin/pilot-agent
+
+# pilot-agent and envoy may run with effective uid 0 in order to run envoy with
+# CAP_NET_ADMIN, so any iptables rule matching on "-m owner --uid-owner
+# istio-proxy" will not match connections from those processes anymore.
+# Instead, rely on the process's effective gid being istio-proxy and create a
+# "-m owner --gid-owner istio-proxy" iptables rule in istio-iptables.sh.
+RUN \
+  chgrp 1337 /usr/local/bin/envoy /usr/local/bin/pilot-agent && \
+  chmod 2755 /usr/local/bin/envoy /usr/local/bin/pilot-agent
+
+ADD envoy_pilot.yaml.tmpl /etc/istio/proxy/envoy_pilot.yaml.tmpl
+ADD envoy_policy.yaml.tmpl /etc/istio/proxy/envoy_policy.yaml.tmpl
+ADD envoy_telemetry.yaml.tmpl /etc/istio/proxy/envoy_telemetry.yaml.tmpl
+ADD istio-iptables.sh /usr/local/bin/istio-iptables.sh
+
+COPY envoy_bootstrap_v2.json /var/lib/istio/envoy/envoy_bootstrap_tmpl.json
+
+ENV ISTIO_META_ISTIO_PROXY_SHA $proxy_version
+ENV ISTIO_META_ISTIO_VERSION $istio_version
```

Comment:
These are outdated? ISTIO_META_ISTIO_PROXY_VERSION is what we read in Pilot IIRC

Reply:
Those are additional variables.